packages/api/src/middleware/auth.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from "express";
+
+const isAuthenticated = function (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) {
+  if (req.isAuthenticated()) {
+    return next();
+  }
+  res.status(401).send("Unauthenticated user").redirect("/login");
```

Comment:
I don't think we need the redirect here because the frontend and backend are separate
Also I think we should do `.send({ message: "Unauthorized" })` or something to be consistent with other places we send errors